spec/models/picture_spec.rb

```diff
@@ -1,15 +1,25 @@
 describe Picture do
   subject { FactoryGirl.build :picture }
 
+  before do
```

Comment:
Did you consider doing `subject { FactoryGirl.build(:picture, :content => "foo") }` above?